examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -305,6 +314,11 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback {
         }
     }
 
+    private fun initDynamicCamera() {
```

Comment:
for init dynamic camera need route, so may be 
```suggestion
    private fun initDynamicCamera(route: DirectionRoute) {
```

Reply:
done